examples/zookeeper.py

```diff
@@ -17,9 +17,8 @@ def __init__(self):
 
 		# Advertise self thru ZooKeeper
 		zksvc = self.get_service("asab.ZooKeeperService")
+		zksvc.DefaultContainer
```

Comment:
This looks unfinished. You could add an example of writing/reading zookeeper data here. Or at least a TODO note.

Reply:
I am working on this now!